src/Dashboard/StakePoolInfos.tsx

```diff
@@ -66,7 +72,7 @@ export const useStakingInfo = (wallet: YoroiWallet) => {
         poolId: stakingStatus.poolKeyHash,
         amount,
         rewards: accountState.remainingAmount,
-      }
+      } as Staked
```

Comment:
can we ->
```
const staked: Staked = {
        status: 'staked',
        poolId: stakingStatus.poolKeyHash,
        amount,
        rewards: accountState.remainingAmount,
      }

return staked
```

to avoid casting?

Reply:
what we want is `{ type: "registered" } satisfies Registered`
but that doesnt come out until typescript 4.9

Reply:
although, casting doesnt change anything here, and its still type safe